strider/query_planner.py

```diff
@@ -390,8 +407,14 @@ async def generate_plans(
                 # Find edges that get us from
                 # There could be multiple edges that need to each
                 # be added as a separate step
-                step = Step(edge['subject'], edge_id, edge['object'])
                 op = get_operation(current_qg, edge)
+
+                # Reverse edges don't actually exist, so the steps in the plan
+                # should just refer to the forward edges
+                if edge_id.endswith(REVERSE_EDGE_SUFFIX):
+                    edge_id = edge_id.replace(REVERSE_EDGE_SUFFIX, '')
```

Comment:
Is the suffix just to prevent key collisions in `expanded_qg["edges"]`? It feels like there should be a neater way to do this...

Reply:
Yeah it is. I can't really think of a neater way off the top of my head. We could store the reverse edges in a separate list, but that would be more messy. This way is nice because it lets us pretty easily handle the reverse edges when building the plan as well.

Right now this isn't bulletproof. Sending a query graph with a node key of 'e01-reversed' will break things. We could modify this to be a little better at handling that case, at the cost of code complexity.